pact/ledger.pact

```diff
@@ -200,11 +200,10 @@
 
   (defun create-account:bool
     ( id:string
-      account:string
+      ; account:string ? create account from (create-principal )
```

Comment:
I personally would opt for leaving the account as input param to the function, for the following reasons:

- The builders have to call `create-principal` themselves which increases awareness of account principals in general
- It is in line with other functions in the ledger, like `ledger.create-token` which takes in a token-id that is validated in the function as well.